.gitignore

```diff
@@ -0,0 +1,5 @@
+'.idea' 
+
+'git push --set-upstream origin insertsort' 
+'sortproject.xml' 
+'.idea','target','sortproject.iml' 
```

Comment:
You could go for something like `*.iml`
`.idea` is duplicated at line 1

Reply:
done